src/hooks/useApproveCallback.ts

```diff
@@ -99,16 +110,26 @@ export function useApproveCallback(
 }
 
 // wraps useApproveCallback in the context of a swap
-export function useApproveCallbackFromTrade(trade?: UniswapV2Trade /* allowedSlippage = 0 */) {
+export function useApproveCallbackFromTrade(trade?: Trade /* allowedSlippage = 0 */) {
```

Comment:
Undesired comment

```suggestion
export function useApproveCallbackFromTrade(trade?: Trade) {
```

Reply:
I'll ignore this one unless you tell me otherwise